nautobot/core/tests/test_authentication.py

```diff
@@ -157,39 +158,46 @@ def test_EXTERNAL_AUTH_DEFAULT_groups(self):
         self.assertSetEqual({groups[0], groups[1]}, set(new_user.groups.all()))
 
     @override_settings(
-        AUTHENTICATION_BACKENDS=TEST_AUTHENTICATION_BACKENDS,
-        REMOTE_AUTH_AUTO_CREATE_USER=True,
-        EXTERNAL_AUTH_DEFAULT_PERMISSIONS={
-            "dcim.add_site": None,
-            "dcim.change_site": None,
-        },
+        SOCIAL_AUTH_BACKEND_PREFIX="custom_auth.backend",
     )
-    def test_external_auth_default_permissions(self):
```

Comment:
What was the reason for deleting this existing test case?

Reply:
I didn't mean to so I'll be adding it back shortly.